### PR TITLE
Android O

### PR DESCRIPTION
**Pull request by danielpassos**
Related issues:

* [FH-4340](https://issues.jboss.org/browse/FH-4340) - Update helloworld-android-gradle to Android O

### CODE REVIEW

File: app/src/main/java/com/feedhenry/helloworld/HelloFragment.java [hunk 1]
```diff
@@ -67,7 +67,7 @@ private void cloudCall(final View button, final String name, final TextView resp
                 public void success(FHResponse fhResponse) {
                     Log.d(TAG, "cloudCall - success");
                     button.setEnabled(true);
-                    response.setText(fhResponse.getJson().getString("msg"));
+                    response.setText(fhResponse.getJson().getString("text"));
```
Comment (wojta) [inline app/src/main/java/com/feedhenry/helloworld/HelloFragment.java]: I'm not sure about this, it runs against helloworld-cloud and there is still "msg".
https://github.com/feedhenry-templates/helloworld-cloud/blob/master/lib/hello.js#L17

Comment (jcesarmobile) [reply app/src/main/java/com/feedhenry/helloworld/HelloFragment.java]: Yeah, app crashes because of this

**Review by jcesarmobile — CHANGES_REQUESTED** [review]
Response is msg, not text, it should be changed, the app crash when calling the cloud app

File: app/src/main/java/com/feedhenry/helloworld/HelloFragment.java (hunk 1 above)
Comment (danielpassos) [reply app/src/main/java/com/feedhenry/helloworld/HelloFragment.java]: Hrm, I did not touched on it, but I have commited a fix

**Review by jcesarmobile — APPROVED** [review]
LGTM

**Pull request merged**